content/about/use-case/_index.en.md

```diff
@@ -1,20 +1,100 @@
 ---
 title: Use cases
 linkTitle: Use cases
-description: Use cases and strategic axes
+description: Use cases related to rail transport planning
 weight: 60
 ---
 
-### OSRD at SNCF Réseau, the French railway infrastructure manager
+![OSRD presentation](osrd_presentation.png)
 
-The process of planning the circulation of a train requires **10 years of anticipation** and involves the collaboration of different services and actors to transform the mobility needs of customers: **organizing authorities (OA)** and **railway companies (RC)** into **annual service (AS)**.
+OSRD is meant to fulfill a range of use-cases related to railway planning:
+ - [timetabling](#timetabling)
+ - [operation studies](#%C3%A9tudes-dexploitation)
+ - [infrastructure management](#infrastructure-management)
+ - [short term capacity management](#short-term-capacity-management)
 
-The OSRD project aims to meet the needs of simulations that cover the entire process, from the design of the transportation plan to its operation.
 
-![OSRD presentation](osrd_presentation.png)
+![produits OSRD](osrd_product.en.png)
+
+
+## Timetabling
+
+Timetabling is about designing a transport service in order
+to meet the transit needs.
+
+OSRD addresses this by providing:
+ - a railway infrastructure database at microscopic scale
+ - a rolling stock database
+ - an infrastructure edition tool
+ - a timetabling tool
+ - train path allowances, including an allowance distribution which minimizes energy use
+
+Planned features include:
+ - automated conflict detection[^conflict]
+ - macroscopic infrastructure synthesis from microscopic infrastructure
+ - interactive timetable diagrams
+ - a throughput and rolling stock turnover optimizer
+
+[^conflict]: A conflicts occurs when the movement of a train interferes with another
+
+
+## Operation studies
+
+Operation studies assess the ability of infrastructures to meet demand,
+and balance infrastructure capacity with transportation needs.
+They enable informed public infrastructure investment decisions.
+
+OSRD addresses this need by providing:
+ - a railway infrastructure database at microscopic scale
+ - a rolling stock database
+ - an infrastructure edition tool
+ - a timetabling tool
+ - train path allowances, including an allowance distribution which minimizes energy use
+ - a signaling simulator
+
+Planned features include:
+ - automated conflict detection[^conflict]
+ - automated timetable robustness assessment using stochastic simulation
+ - automated throughput analyzer
+
+
+## Infrastructure management
+
+Processing railway infrastructure data is a requirement shared by all of OSRD's features.
+Therefore, a dedicated module provides:
+ - a customizable infrastructure map
+ - an infrastructure editor
+ - a search feature
+ - an import tool[^railjson]
```

Comment:
No export tool?

Reply:
Yes of course ! But one day,  in different formats :)